docs/_doc/tutorials/wurst_for_vjass_users/page_1.md

```diff
@@ -0,0 +1,234 @@
+---
+title: Wurst for vJass Users
+sections:
+- Why not vJass?
+- Why wurst?
+- On performance
+- Is vJass bad then?
+- Is wurst good?
+- Summary
+- FAQs
+---
+
+{: .answer}
+### *&nbsp;*{: .fa .fa-exclamation-circle} This guide expects the reader to be familiar with vJass.
+
+{: .answer}
+### *&nbsp;*{: .fa .fa-exclamation-circle} This tutorial is in the format of an informal discussion, but many of the key points are backed by well understood, high-level programing paradigms.
+------
+
+When I ~evangelize~ discuss wurst, I tend to hear weird arguments against using it.
+Some of them are intereting, like "well I can't justify learning it", or "my map is already using vJass", but other arguments tend to be outlandish and worth some education.
+More concretely, lots of arguments about wurst are spoilt by misunderstanding or misinformation.
+
+For what it's worth, wurst is an easy language to learn, and you can write wurst in a map that use vJass, but I digress.
+
+As background: I switched to using wurst in the last couple years after fairly extensive experience writing vJass.
+The switch was easy - I decide to "try" wurst on a toy map, and immediately realized that I had blinders on, and indeed was *misled* by the public opinion of others.
+
+So this tutorial is about learning, but it's also about justice.
+
+May the most correct man win - not the loudest.
+
+## Why not vJass?
+
+This is an argument that wurst users have historically done poor at.
+The most interesting problems with vJass are not silly quirks like "extends array" or the macro system - no one actually cares about those things, unless you're a language purist.
```

Comment:
Hivers were criticizing wurst for not having such a construct as _extends array_, hence it became a running gag.
It's not that noone cares - but vJass having such a **bad** language feature and it being demanded by users that have gotten used to "hacking" their code together instead of actually relying on good language design.
I think this should be reflected here a bit